### ISSUE TITLE
AttributeError: can't set attribute 'showHiddenFiles' in app.py#1276

### ISSUE THREAD

**Issue by aryoda** [bit-team/backintime#1532]
During testing of the current dev version when preparing the new release I have found a bug when clicking the button "**Show hidden files**" in the GUI:

```
Traceback (most recent call last):
  File "/home/<user>/dev/backintime/qt/app.py", line 1280, in btnShowHiddenFilesToggled
    self.showHiddenFiles = checked
AttributeError: can't set attribute 'showHiddenFiles'
```

https://github.com/bit-team/backintime/blob/772ac55a80c90564591bdaaeaf9e59d0b53fcb27/qt/app.py#L1275-L1277

It seems during the GUI refactoring this attribute has been changed into a `@property` but without a setter (so it is read-only).

https://github.com/bit-team/backintime/blob/772ac55a80c90564591bdaaeaf9e59d0b53fcb27/qt/app.py#L418-L420

**Comment by aryoda** [comment]
This is a regression, it worked before the GUI refactoring.

The actual status of the "show hidden files" button is saved in the config file if the GUI is closed, see the `closeEvent()`:

https://github.com/bit-team/backintime/blob/772ac55a80c90564591bdaaeaf9e59d0b53fcb27/qt/app.py#L789

https://github.com/bit-team/backintime/blob/772ac55a80c90564591bdaaeaf9e59d0b53fcb27/qt/app.py#L803
So my proposal is to fix this by introducing a setter using this code:

```
# TODO The qt.show_hidden_files key should be a constant instead of a duplicated string
@showHiddenFiles.setter
def showHiddenFiles(self, value):
    self.config.setBoolValue('qt.show_hidden_files', value)
```

Then setting this value in the `closeEvent()` is no longer required.

**Comment by buhtz** [comment]
I agree. The proposal sounds fine to me.

**Comment by aryoda** [comment]
> I agree. The proposal sounds fine to me.

It is contained as fix in the PR #1533 and tested (does work)
